### PR TITLE
suggestions and edits

### PR DESCRIPTION
**Pull request by iAmMichaelConnor**
Hi Youssef, I've made some suggestions for the EIP document, including updates to make byte-lengths multiples of 32.

### CODE REVIEW

File: EIPS/eip-xxxx.md [hunk 1]
```diff
@@ -15,34 +15,52 @@ requires : xxxx
...
-- G1MULTIEXP - to perform multiexponentiation on a curve defined over prime field
-- G2ADD - to perform point addition on a curve twist defined over quadratic extension of the base field
-- G2MUL - to perform point multiplication on a curve twist defined over quadratic extension of the base field
-- G2MULTIEXP - to perform multiexponentiation on a curve twist defined over quadratic extension of the base field
-- PAIRING - to perform a pairing operations between a set of *pairs* of (G1, G2) points
+- BW6_G1_ADD - to perform point addition on a curve defined over prime field
+<!--TODO: consider removing BW6_G1_MUL-->
+- BW6_G1_MUL - to perform point multiplication on a curve defined over prime field
+- BW6_G1_MULTIEXP - to perform multiexponentiation on a curve defined over prime field
+- BW6_G2_ADD - to perform point addition on a curve twist defined over quadratic extension of the base field
```
Comment (yelhousni) [inline EIPS/eip-xxxx.md]:
```suggestion
- BW6_G2_ADD - to perform point addition on a curve twist defined the base prime field
```

File: EIPS/eip-xxxx.md [hunk 2]
```diff
@@ -15,34 +15,52 @@ requires : xxxx
...
+<!--TODO: consider removing BW6_G2_MUL-->
+- BW6_G2_MUL - to perform point multiplication on a curve twist defined over quadratic extension of the base field
```
Comment (yelhousni) [inline EIPS/eip-xxxx.md]:
```suggestion
- BW6_G2_MUL - to perform point multiplication on a curve twist defined over the base prime field
```

File: EIPS/eip-xxxx.md [hunk 3]
```diff
@@ -15,34 +15,52 @@ requires : xxxx
...
+- BW6_G2_MULTIEXP - to perform multiexponentiation on a curve twist defined over quadratic extension of the base field
```
Comment (yelhousni) [inline EIPS/eip-xxxx.md]:
```suggestion
- BW6_G2_MULTIEXP - to perform multiexponentiation on a curve twist defined over the base prime field
```

File: EIPS/eip-xxxx.md [hunk 4]
```diff
@@ -15,34 +15,52 @@ requires : xxxx
...
+- BW6_PAIRING - to perform a pairing operations between a set of *pairs* of (G1, G2) points
 
-Multiexponentiation operation is included to efficiently aggregate public keys or individual signer's signatures during BLS signature verification, as well as public inputs in SNARKs.
+The multiexponentiation operations are a generalization of point multiplication.
 
 ## Motivation
 <!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
 
-Motivation of this precompile is to allow efficient one layer composition of SNARK proofs. Currently this is done by Zexe using the BW6-761/SW6 pair of curves. This precompile proposes a replacement of SW6 by BW6-761 which allows five times faster verification.
+The motivation of this precompile is to allow efficient one-layer composition of SNARK proofs. Currently this is done by Zexe using the BLS12-377/SW6 pair of curves. This precompile proposes a replacement of SW6 by BW6-761, which allows five times faster verification.
```
Comment (yelhousni) [inline EIPS/eip-xxxx.md]:
```suggestion
The motivation of this precompile is to allow efficient one-layer composition of SNARK proofs. Currently this is done by Zexe using the BLS12-377/SW6 pair of curves. This precompile proposes a replacement of SW6 by BW6-761, which allows much faster verification.
```

Comment (yelhousni) [reply EIPS/eip-xxxx.md]: It seems that the speedup factor varies significantly from an implementation to another.

**Review by yelhousni — CHANGES_REQUESTED** [review]
@iAmMichaelConnor, LGTM. I just fixed an erratum I had in G2 precompiles description.

**Comment by iAmMichaelConnor** [comment]
Thanks for those suggestions Youssef; I've committed those changes.

**Pull request merged**